modules/nw-modifying-operator-install-config.adoc

```diff
@@ -28,7 +28,9 @@ parameter.
 
 [IMPORTANT]
 ====
-Modifying the {product-title} manifest files directly is not supported.
+Modifying the {product-title} manifest files created by the installation program
+is not supported. Applying a manifest file that you create, as in the following
```

Comment:
The text looks good to me, but the lines should not be hard wrapped. See here in our guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

Reply:
Thank you Andrea!